dockerfiles/README.md

```diff
@@ -0,0 +1,10 @@
+# Obscuro enclave service Docker image
+
+To create the Docker image for an Obscuro enclave service running in SGX simulation mode:
+
+    docker build -t obscuro_enclave - < dockerfiles/enclave
+
+To run the image as a container, where `XXXX` is the local port on which to expose the enclave and `YYY` is the node 
```

Comment:
as a user of this, it's not clear what I should use for XXXX and YYY and how it matters for running the test

Reply:
Will try and clarify. NB, these are supposed generic instructions, not test specific.